### PR TITLE
Update tweet_nouns.py

### PR DESCRIPTION
**Pull request by GauravKrv**
Added 10 nouns to #food, 3 nouns to #computer things, and a new key "#dance" with 10 nouns

### CODE REVIEW

File: words_dir/tweet_nouns.py [hunk 1]
```diff
@@ -200,7 +200,17 @@
     'vada pav',
     'samosa',
     'jalebi',
-    'cadbury🍫',
+    'cadbury🍫'
+    'biriyani'
+    'samosa'
```
Comment (VickiLanger) [inline words_dir/tweet_nouns.py]: This is already on line 201

**Review by VickiLanger — CHANGES_REQUESTED** [review]
Thanks for adding all of these. Please take a look and make sure you have commas after each item. If the comma isn't there, it will concatenate or smash together the adjacent strings.

File: words_dir/tweet_nouns.py [hunk 2]
```diff
@@ -200,7 +200,17 @@
...
+    'chutney'
+    'dosa'
+    'maal-pua'
+    'katchori'
+    'kheer'
+    'chowmin'
+    'tea'
```
Comment (GauravKrv) [inline words_dir/tweet_nouns.py]: Comas have been missed out in these lines

**Review by GauravKrv — COMMENTED** [review]
Resolved the coma isue;)

**Review by VickiLanger — APPROVED** [review]
Looks good. Thanks for your contribution.

---
Don't forget to follow the bot on [Twitter](twitter.com/lgbtqotd) to see the hard work you put in.

**Pull request merged**